common/bitvec/src/lib.rs

```diff
@@ -75,6 +79,25 @@ impl BitVec {
         let bucket_pos = pos as usize - (bucket * BUCKET_SIZE);
         (self.inner[bucket] & (0x01 << bucket_pos)) != 0
     }
+
+    // TODO(kostas): Remove after applying it to multi-sig.
+    #[allow(dead_code)]
+    /// Returns the number of set bits.
+    pub fn count_ones(&self) -> u32 {
```

Comment:
since the maximum number of set bits is limited by u8, we can return u8 here too. Alternatively, if you want to support setting bits over u8::max, we should change the `set` and `is_set` functions to accept u32.

Reply:
So the reason I used `u32` is because if all bits are set we get 256 ones and this does not fit to u8.

Reply:
absolutely right. My bad!